blog/admin.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import Group
+
+from blog.models import Post, Commentary, User
+
+
+@admin.register(User)
+class UserAdmin(UserAdmin):
```

Comment:
The class name 'UserAdmin' is already used for the imported 'UserAdmin' from 'django.contrib.auth.admin'. Consider renaming this class to avoid confusion or potential conflicts.